pkg/appgateserver/sdkadapter/relay_client.go

```diff
@@ -0,0 +1,72 @@
+package sdkadapter
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"net/http"
+	"net/url"
+
+	"cosmossdk.io/depinject"
+	"github.com/pokt-network/shannon-sdk/sdk"
+)
+
+var _ sdk.RelayClient = (*sdkRelayClient)(nil)
+
+// sdkRelayClient is a wrapper around the http.Client that implements the
```

Comment:
The "wrapper like" comments are too much implementation info IMO. Just explain what the struct does.